sdk/cosmos/azure-cosmos-spark_3_2-12/src/test/scala/com/azure/cosmos/spark/SparkE2EChangeFeedSplitITest.scala

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.cosmos.spark
+
+import com.azure.cosmos.CosmosClientBuilder
+import com.azure.cosmos.implementation.{TestConfigurations, Utils}
+import com.azure.cosmos.models.ThroughputProperties
+import com.azure.cosmos.spark.diagnostics.BasicLoggingTrait
+import org.apache.hadoop.fs.Path
+import org.scalatest.Ignore
+
+import java.nio.file.Paths
+import java.util.UUID
+
+@Ignore
```

Comment:
You can remote ignore I am skipping the test in emulator below

Reply:
removed